src/test/kotlin/org/rust/lang/core/resolve/RsResolveTest.kt

```diff
@@ -1330,4 +1332,54 @@ class RsResolveTest : RsResolveTestBase() {
                      //^
         }
     """)
+
+    // During our tests, #[cfg(test)] evaluates to true
+    fun `test fn cfg`() = checkByCode("""
```

Comment:
Let's move these tests to a separate class like `RsCfgAttrResolveTest`.
Also, let's add tests with `cfg`ed mod decl, inline mod, use item, extern crate (with alias), macro call that defines some items